homeassistant/components/vicare/strings.json

```diff
@@ -424,6 +424,33 @@
       },
       "compressor_phase": {
         "name": "Compressor phase"
+      },
+      "ventilation_demand": {
+        "name": "Ventilation demand",
+        "state": {
+          "ventilation": "Ventilation"
+        }
+      },
+      "ventilation_level": {
+        "name": "Ventilation level",
+        "state": {
+          "standby": "Standby",
+          "levelone": "1",
+          "leveltwo": "2",
+          "levelthree": "3",
+          "levelfour": "4"
+        }
+      },
+      "ventilation_reason": {
+        "name": "Ventilation reason",
+        "state": {
+          "standby": "Standby",
```

Comment:
Standby is a common string
`[%key:common::state::standby%]`

Reply:
```suggestion
          "standby": "Standby",
          "levelone": "1",
          "leveltwo": "2",
          "levelthree": "3",
          "levelfour": "4"
        }
      },
      "ventilation_reason": {
        "name": "Ventilation reason",
        "state": {
          "standby": "[%key:common::state::standby%]",
```